apps/test/unit/templates/manageStudents/ManageStudentsTableTest.js

```diff
@@ -28,6 +28,7 @@ import teacherSections, {
 import sectionData, {setSection} from '@cdo/apps/redux/sectionDataRedux';
 import scriptSelection from '@cdo/apps/redux/scriptSelectionRedux';
 import isRtl from '@cdo/apps/code-studio/isRtlRedux';
+import NoSectionCodeDialog from '../../../../src/templates/manageStudents/NoSectionCodeDialog';
```

Comment:
NIT: If you'd like, another way to write this is '@cdo/apps/templates/manageStudents/NoSectionCodeDialog'